clients/src/main/java/org/apache/kafka/common/requests/CreateTopicsResponse.java

```diff
@@ -60,6 +61,10 @@ public String messageWithFallback() {
             return message;
         }
 
+        public ApiException exception() {
+            return error.exception(message);
```

Comment:
Hmm, good catch.  I think it makes more sense to add the fallback logic into `Errors#exception`, though.  If null is passed to that method, it should return an exception with the default message for that error code.